Makefile

```diff
@@ -250,7 +250,7 @@ proto-all: proto-format proto-lint proto-gen
 
 proto-gen:
 	@echo "Generating Protobuf files"
-	$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace tendermintdev/sdk-proto-gen:v0.3 sh ./scripts/protocgen.sh
+	$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace tendermintdev/sdk-proto-gen:v0.7 sh ./scripts/protocgen.sh
```

Comment:
It seems like we are getting some proto buf breakage in this PR. I remember playing around with the docker image back in June and upgrading above v. 0.3  broke stuff. It might be worth reverting this to 0..3 and testing again to figure out if it fixes our test. :)

Reply:
tendermintdev/sdk-proto-gen:v0.7 uses buf binary that support "buf generate", prior version doesn't have this feature.  I haven't updated this to main yet so there maybe some outdated script

Reply:
Thats my point. I experienced some breakage in the underlying protobuf "code" after updating from protogogo to buf generate. So I ended up reverting to 0.3. If we can make it work with 0.7 we should update to that since protogogo is deprecated for a while now, however since it was still working I  just decided to  not spend effort on it when I looked at it 6  months ago :)